### PR TITLE
Fix Jakarta Rest TCK issues when running EE10

### PR DESCRIPTION
**Pull request by jim-krueger**

### CODE REVIEW

**Comment by jim-krueger** [comment]
#run-libby-bot

**Comment by jim-krueger** [comment]
#build

**Comment by jim-krueger** [comment]
#build

**Comment by jim-krueger** [comment]
#run-libby-bot

File: dev/io.openliberty.jakarta.rest.3.1.internal_fat_tck/publish/tckRunner/tck/pom.xml [hunk 1]
```diff
@@ -88,9 +88,9 @@
     <dependencyManagement>
         <dependencies>
             <dependency>
-                <groupId>jakarta.platform</groupId>
-                <artifactId>jakarta.jakartaee-api</artifactId>
-                <version>${jakartaee-api.version}</version>
```
Comment (jhanders34) [inline dev/io.openliberty.jakarta.rest.3.1.internal_fat_tck/publish/tckRunner/tck/pom.xml:93]: so you are no long using it should jakartaee-api.version property above be removed as well?

File: dev/io.openliberty.jakarta.rest.3.1.internal_fat_tck/publish/tckRunner/tck/pom.xml [hunk 2]
```diff
@@ -323,12 +323,16 @@
                         <dependency>io.openliberty.jakarta.ws.rs:jakarta-restful-ws-tck</dependency>
                     </dependenciesToScan>
                     <includes>
+                        <!--  Example to run just one test. -->
+                        <include>ee/jakarta/tck/ws/rs/spec/contextprovider/JsonbContextProviderIT</include>
```
Comment (jhanders34) [inline dev/io.openliberty.jakarta.rest.3.1.internal_fat_tck/publish/tckRunner/tck/pom.xml:326]: did you want this left in?

**Review by jhanders34 — APPROVED** [review]

File: dev/io.openliberty.jakarta.rest.3.1.internal_fat_tck/publish/tckRunner/tck/pom.xml (hunk 2 above)
Comment (jim-krueger) [reply dev/io.openliberty.jakarta.rest.3.1.internal_fat_tck/publish/tckRunner/tck/pom.xml:326]: Nope.  Will remove thanks

File: dev/io.openliberty.jakarta.rest.3.1.internal_fat_tck/publish/tckRunner/tck/pom.xml (hunk 1 above)
Comment (jim-krueger) [reply dev/io.openliberty.jakarta.rest.3.1.internal_fat_tck/publish/tckRunner/tck/pom.xml:93]: Yes.  done

**Comment by jim-krueger** [comment]
#run-libby-bot

**Comment by jim-krueger** [comment]
#build

**Comment by jim-krueger** [comment]
#build

**Comment by jim-krueger** [comment]
#build

**Pull request merged**